_data/comunidades/comunidad-madrid.json

```
@@ -47,10 +47,15 @@
       "name": "Boletín oficial Comunidad Madrid",
       "twitter": "ComunidadMadrid"
     },
+    {
+      "url": "cjcm.es",
+      "name": "Consejo de la Juventud de la Comunidad de Madrid",
+      "twitter": "ComunidadMadrid"
+    },
     {
       "url": "www.madrid112.es",
       "name": "112 Emergencias Madrid",
-      "twitter": "112cmadrid"
+      "twitter": "_CJCM_"
```

El correcto es el que estaba, creo que se te ha "traspapelao"

```suggestion
      "twitter": "112cmadrid"
```